templates/rustdoc/header.html

```diff
@@ -161,7 +161,7 @@
                                         <ul class="pure-menu-list">
                                             <li>
                                                 <a href="{{ crate_url | safe }}" class="pure-menu-link">
-                                                    <b>{{ percent | round(precision=2) }}%</b>
+                                                    <b>{{ percent | round(precision=2 )}}%</b>
```

Comment:
I don't think you meant to change this.
```suggestion
                                                    <b>{{ percent | round(precision=2) }}%</b>
```

Reply:
Good catch!